app/src/display/widgets/Kconfig

```diff
@@ -1,6 +1,12 @@
 
 menu "ZMK Display Widgets"
 
+config ZMK_WIDGET_LAYER_STATUS
+    bool "Widget for highest, active layer using small icons"
+    default y
+    depends on !ZMK_SPLIT_PERIPHERAL
+    select LVGL_BUILD_IN_FONT_ROBOTO_12
```

Comment:
Is this actually the font? This may need to be rebased for Zephyr 2.4.0 changes, since I think the font selections there changed.

Reply:
Yes, it's a font ; I swapped it to be more consistent with the other font in use and ensured it looks good on the oled.